lib/cfer/core/stack.rb

```diff
@@ -155,7 +155,11 @@ def output(name, value, options = {})
     # Renders the stack into a CloudFormation template.
     # @return [String] The final template
     def to_cfn
-      to_h.to_json
+      if @options[:pretty_print]
+        Cfer::Util::Json.format_json(to_h)
```

Comment:
Cool, your comment makes sense. My only comment is to maybe use `JSON.pretty_generate(to_h)` here instead - the Cfer JSON generator uses the Rainbow gem to add color codes for terminal output. I'm not sure how well AWS plays with that.